src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,97 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+  padding-bottom: 150px;
+  padding-left: 75px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 78px;
+    padding-left: 0;
+    padding-top: 50px;
+  }
+}
+
+.allTextWrap {
+  grid-column-end: 11;
+  grid-column-start: 2;
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  margin: 0;
+  padding-bottom: 54px;
+}
+
+.number {
```

Comment:
I see that this is a special typography style that is not part of our normal style guide, which makes sense, since this is a special component and we want something that adds extra emphasis to the statistic number.

However, this means that you should be setting all of the font-related styles. Right now, this is defaulting to the browser default font, which on macOS and Chrome is Times New Roman, but will vary on different browsers and OSs.

We use two typefaces on our site: Merriweather and Lato. Merriweather is a [serifed](https://en.wikipedia.org/wiki/Serif) font, whereas Lato is a [sans-serif](https://en.wikipedia.org/wiki/Sans-serif) font. In the Figma designs, I see that these stat numbers use an entirely different font, [Inter](https://fonts.google.com/specimen/Inter), which is a sans-serif font.

I don't think it's a good idea to include a third font face into our site, since that will make the site slower to load, as it's a third font that needs to be loaded, but we could try use Lato in its place, as an alternative san-serif font.

You should also appropriately set the font weight and line height, which are other properties that we normally have set with our font shorthand variables.

Reply:
Added Lato font and line-height, but according to Figma, font-weight is bold which does not look right.

Reply:
I just left a comment on Figma just now, but I think it's OK to use Lato for now. There's consequences for requiring too many web fonts on a site, since it'll require a user to download all the fonts, so I'd like to stick to the original two fonts, if we can.

Reply:
Also, I don't see the line-height actually here for the non-mobile view.